src/vm/moar/dispatchers.nqp

```diff
@@ -3374,6 +3406,17 @@ nqp::register('raku-invoke', -> $capture {
     my $code_arg := nqp::track('arg', $capture, 0);
     guard-type-concreteness($code_arg) unless $code-constant;
 
+    if nqp::isconcrete(
+        my $required-revision := nqp::decont(nqp::how_nd($code).find_method(nqp::decont($code), 'REQUIRED-REVISION', :no_fallback))
+    ) && $required-revision($code) > nqp::getlexcaller('$?LANGUAGE-REVISION')
```

Comment:
Just a nit: I have personally stopped using any variant of "greater than" comparison operators. If you flip it around you get ascending values from left to right which for left-to-right-readers is a very natural order. Of course that's a matter of personal preference.

Reply:
I've adjusted these cases now.